packages/vite/src/node/plugins/reporter.ts

```diff
@@ -148,9 +148,11 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
                     return {
                       name: chunk.fileName,
                       group: 'JS',
-                      size: chunk.code.length,
+                      size: Buffer.from(chunk.code).length,
```

Comment:
```suggestion
                      size: Buffer.byteLength(chunk.code),
```
`Buffer.byteLength` should be performant than `Buffer.from().length` as it doesn't require the whole converted value to be held in the memory.